iOS/Flows/Home/Members/MemberCell.swift

```diff
@@ -9,24 +9,40 @@
 import Foundation
 
 class MemberCell: CollectionViewManagerCell, ManageableCell {
+
     typealias ItemType = String
     var currentItem: String?
     
     private let personView = BorderedPersonView()
+    private let videoView = VideoView()
     private let label = ThemeLabel(font: .regular, textColor: .whiteWithAlpha)
     
     override func initializeSubviews() {
         super.initializeSubviews()
         
         self.contentView.addSubview(self.personView)
+        // TODO: Layout and size the video view or add it to the person view.
+        self.contentView.addSubview(self.videoView)
         self.contentView.addSubview(self.label)
         self.label.textAlignment = .center
     }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        self.label.setSize(withWidth: self.contentView.width)
+        self.label.pin(.bottom)
+        self.label.centerOnX()
+        
+        self.personView.squaredSize = self.contentView.height - 30
+        self.personView.centerOnX()
+        self.personView.pin(.top)
+    }
 
     func configure(with item: String) {
+        // TODO: Configure the video view.
         Task.onMainActorAsync {
-    
-            guard let person = await PeopleStore.shared.getPerson(withPersonId: item) else { return }            
+            guard let person = await PeopleStore.shared.getPerson(withPersonId: item) else { return }
```

Comment:
We'll want to grab the person's latest moment and then use it to configure the video view. Alternatively we could allow the person view to handle video playback.